cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java

```diff
@@ -18,6 +18,7 @@
     String RUN_ENV = "runEnv";
     String HOOKS = "hooks";
     String NODE_NAME_KEY = "nodeName";
+    String EXECUTABLE_TYPE = "executableType";
```

Comment:
we need to add a dependency to cloudslang-runtime in the cloudslang-compiler pom to do that.